src/mpl/src/shm/mpl_shm.c

```diff
@@ -18,9 +18,7 @@
  */
 int MPL_shm_hnd_serialize(char *str, MPL_shm_hnd_t hnd, int str_len)
 {
-    int rc = -1;
-    rc = MPLI_shm_ghnd_get_by_val(hnd, str, str_len);
-    return rc;
+    return MPLI_shm_ghnd_get_by_val(hnd, str, str_len);
 }
```

Comment:
Should this change go to commit: `mpl/shm: Remove unnecessary variables for return` ?